recipe_templates/pyregion/meta.yaml

```diff
@@ -31,14 +31,16 @@ source:
 requirements:
   build:
     - python
+    - astropy
     - setuptools
     - pyparsing !=2.0.0
-    - numpy x.x   # pin the numpy version, please...
+    - numpy >1.6
```

Comment:
I'm confused -- `requirements.yml` says that pyregion has numpy compiled extensions, but if that is the case `numpy x.x` must be listed as dependency. To also include a version restriction do this:

```
- numpy x.x
- numpy >1.6
```